ts/features/payments/checkout/components/WalletPaymentFailureDetail.tsx

```diff
@@ -50,75 +48,68 @@ const WalletPaymentFailureDetail = ({ failure }: Props) => {
   };
 
   const getPropsFromFailure = ({
-    faultCodeCategory,
-    faultCodeDetail
+    faultCodeCategory
   }: WalletPaymentFailure): OperationResultScreenContentProps => {
     switch (faultCodeCategory) {
-      case FaultCategoryEnum.PAYMENT_UNAVAILABLE:
+      case "PAYMENT_UNAVAILABLE":
```

Comment:
What about using in the switch-case the `FaultCategoryEnum` enumerator? I noticed that it used to be like that and then they were changed to strings, there is a specific impediment that brought you to this choice?

Reply:
Yes, there is a type conflict. To use the enum we would need to import the `FaultCategoryEnum` exported from every single failure type.